testserver/testserver.go

```diff
@@ -252,13 +252,13 @@ func NewTestServer(opts ...TestServerOpt) (TestServer, error) {
 	// which get us over the socket filename length limit.
 	baseDir, err := ioutil.TempDir("/tmp", "cockroach-testserver")
 	if err != nil {
-		return nil, fmt.Errorf("could not create temp directory: %s", err)
+		return nil, fmt.Errorf("testserver: could not create temp directory: %s", err)
```

Comment:
we should wrap the error here rather than use fmt.Errorf

see https://cockroachlabs.atlassian.net/wiki/spaces/CRDB/pages/1676640931/Error+handling+basics
(although for this library we cannot use cockroachdb/errors, so we can use the golang errors package